Modules/Filtering/FFT/include/itkVnlFFTImageFilterFactory.h

```diff
@@ -0,0 +1,70 @@
+/*=========================================================================
+ *
+ *  Copyright NumFOCUS
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0.txt
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ *=========================================================================*/
+#ifndef itkVnlFFTImageFilterFactory_h
+#define itkVnlFFTImageFilterFactory_h
+#include "ITKFFTExport.h"
+
+#include "itkObjectFactoryBase.h"
+
+namespace itk
+{
+/**
+ *\class VnlFFTImageFilterFactory
+ * \brief Create instances of Vnl FFT ImageFilter objects using an object factory.
+ * \ingroup ITKFFT
+ */
+class ITKFFT_EXPORT VnlFFTImageFilterFactory : public ObjectFactoryBase
+{
+public:
+  ITK_DISALLOW_COPY_AND_MOVE(VnlFFTImageFilterFactory);
+
+  /** Standard class type aliases. */
+  using Self = VnlFFTImageFilterFactory;
+  using Superclass = ObjectFactoryBase;
+  using Pointer = SmartPointer<Self>;
+  using ConstPointer = SmartPointer<const Self>;
+
+  /** Class methods used to interface with the registered factories. */
+  const char *
+  GetITKSourceVersion() const override;
+
+  const char *
+  GetDescription() const override;
+
+  /** Method for class instantiation. */
+  itkFactorylessNewMacro(Self);
+
+  /** Run-time type information (and related methods). */
+  itkTypeMacro(VnlFFTImageFilterFactory, ObjectFactoryBase);
+
+  /** Register one factory of this type  */
+  static void
+  RegisterOneFactory()
+  {
+    auto factory = VnlFFTImageFilterFactory::New();
+
+    ObjectFactoryBase::RegisterFactoryInternal(factory);
```

Comment:
Per above discussion, this line has no practical effect as `VnlFFTImageFilterFactory` never defines its own filter overrides via `this->RegisterOverride`. Instead override functionality is outsourced to `itk::FFTImageFilterFactory`.